tika-core/src/main/resources/org/apache/tika/mime/tika-mimetypes.xml

```diff
@@ -4694,6 +4720,18 @@
   <mime-type type="audio/vnd.vmx.cvsd"/>
   <mime-type type="audio/vorbis-config"/>
   <mime-type type="audio/x-aac">
+<!--TIKAFIX by jerni-zu393: magic bits for "aac" file is added here-->
+  <magic priority="50">
+  <match value="P" type="string" offset="2">
+  <match value="libfaac" type="string" offset="11"/>
+  </match>
+  </magic>
```

Comment:
where is this detection coming from? I cannot find it the same in the `file` util nor looking around.. my AAC file starts with FFF1 which is kind of indicated in https://www.garykessler.net/library/file_sigs.html as an example